lib/prawnto/template_handlers/partials.rb

```diff
@@ -8,11 +8,16 @@ module Partials
       # :prawn_object - The object to use for the pdf object in the partial.
       #                 Defaults to the pdf document, but can take a paramenter to render within a prawn object. This
       #                 is good for items like tables, text_blocks, etc.
-      def partial!(partial_name, prawn_object = pdf)
+      def partial!(partial_name, prawn_object = pdf, locals: {})
```

Comment:
Not sure which ruby versions you want to maintain support for, so this might need to change away from kwarg syntax